core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -41,7 +41,7 @@ THE SOFTWARE.
           <j:if test="${!it.building}">
             ${%Took} <a href="${rootURL}/${it.parent.url}buildTimeTrend">${it.durationString}</a>
           </j:if>
-          <j:if test="${!empty(app.slaves) or (it.builtOnStr != null and it.builtOnStr != '')}">
```

Comment:
Actual production code change: Seems correct, replaces `Hudson#getSlaves()` with `Jenkins#getNodes()`. The former is a deprecated wrapper around the latter.